modules/ROOT/pages/choice-router-concept.adoc

```diff
@@ -85,6 +87,50 @@ http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/htt
 </mule>
 ----
 
+== Adding Additional Route Options
+
+By default, the Choice router has one route option and a default route. To add additional routes to the Choice router, follow these steps:
+
+. From *Mule Palette* in Anypoint Studio, click and hold any component.
+. Drag the component inside the Choice router where you see a vertical black line:
```

Comment:
I think we want to avoid "click and hold":
. From *Mule Palette* in Anypoint Studio, click any component and drag it into the Choice router, where you see a vertical black line: